docs/submitting-applications.md

```diff
@@ -48,6 +48,44 @@ Some of the commonly used options are:
 * `application-jar`: Path to a bundled jar including your application and all dependencies. The URL must be globally visible inside of your cluster, for instance, an `hdfs://` path or a `file://` path that is present on all nodes.
 * `application-arguments`: Arguments passed to the main method of your main class, if any
 
+Alternatively, for submitting on yarn, 
```

Comment:
yarn -> YARN
I don't know if this helps as it's just presenting *both* ways to set this, and not clarifying which is preferred. It also duplicates the rest of the example.